tools/node_config/subnet/topos/genesis.json

```diff
@@ -0,0 +1,51 @@
+    "gasLimit": "0x500000",
+    "difficulty": "0x1",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "alloc": {
+      "0x100D617E4392C02B31bdCe650b26b6c0c3E04F95": {
+        "balance": "0x3b9aca00"
+      },
+      "0x92183Cff18A1328E7d791D607589A15d9EeE4bC4": {
+        "balance": "0x3b9aca00"
+      },
+      "0xB4973Cdb10894D1D1547673bD758589034C2BBa5": {
+        "balance": "0x3b9aca00"
+      },
+      "0xC16d83893cB61872206D4e271B813015D3242d94": {
+        "balance": "0x3b9aca00"
+      }
+    },
+    "number": "0x0",
+    "gasUsed": "0x70000",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "baseFee": "0x0",
+    "baseFeeEM": "0x0"
+  },
+  "params": {
+    "chainID": 100,
+    "engine": {
+      "ibft": {
+        "blockTime": 6000000000,
+        "epochSize": 30000,
+        "type": "PoA",
+        "validator_type": "bls"
+      }
+    },
+    "blockGasTarget": 0,
+    "burnContract": null,
+    "burnContractDestinationAddress": "0x0000000000000000000000000000000000000000"
+  },
+  "bootnodes": [
```

Comment:
Even though the tests run through, there is some tweaking to be done around the `bootnodes`. I assume we need to adjust this to just one `bootnode`, and maybe give it a fixed `PeerId` in the `init.sh`, and replace the list here. cc: @Freyskeyd